_data/simple-icons.json

```diff
@@ -1648,6 +1648,11 @@
             "source": "https://atlassian.design/resources/logo-library",
             "guidelines": "https://atlassian.design/foundations/logos"
         },
+        {
+            "title": "ATS",
+            "hex": "000000",
```

Comment:
This hex is black, you have to pick the color of the brand/logo. See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines.

Reply:
but there are 2 hex values how can i add them

Reply:
![image](https://github.com/user-attachments/assets/399e6214-3649-419e-93bd-d1f1fd21a211)